crates/swc_ecma_transforms_testing/src/lib.rs

```diff
@@ -154,20 +154,20 @@ impl<'a> Tester<'a> {
         name: &str,
         syntax: Syntax,
         src: &str,
-    ) -> Result<Module, ()> {
+    ) -> Result<Program, ()> {
```

Comment:
Is this change necessary? I think we can alternatively change only the argument passed to `fold_**`

Reply:
I need to ensure that what `fold_with` is a program, so the return value has been changed to `Program`.

https://github.com/swc-project/swc/blob/9fce494b10b19ffc21ec99f16a246d71be8a3383/crates/swc_ecma_transforms_testing/src/lib.rs#L180-L182

convert back to Module might be difficult, as it is possible for it to be parsed as a script.
do you have some good ways to improve it?

Reply:
You mean you want to use `parse_program`?
If so, your code is the correct direction

Reply:
Yes, I want to use `visit_mut_program`.
If I get `Module` instead of `Program`, the `visit_mut_program` will be skipped.

See https://github.com/swc-project/swc/pull/7202/files#r1159484231